src/routes/articles.js

```diff
@@ -0,0 +1,104 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+import { assert } from "superstruct";
+import asyncHandler from "../middleWares/errorHandler.js";
+import { CreateArticle, PatchArticle } from "../middleWares/structs.js";
+
+const prisma = new PrismaClient();
+
+const articleRoute = express.Router();
+
+articleRoute
+  .route("/detail")
+  .get(asyncHandler(async (req, res) => {
+      const { offset = 0, limit = 10, order = "newest", search = "" } = req.query;
+      let orderBy;
+      switch (order) {
+        case "oldest":
+          orderBy = { createdAt: "asc" };
+          break;
+        case "newest":
+        default:
+          orderBy = { createdAt: "desc" };
+      }
+      const articles = await prisma.article.findMany({
+        where: {
+          OR: [
+            { title: { contains: search, mode: "insensitive" } },
+            { content: { contains: search, mode: "insensitive" } },
+          ],
+        },
+        orderBy,
+        skip: parseInt(offset),
+        take: parseInt(limit),
+      });
+      res.status(200).send(articles);
+    }))
+
+articleRoute
+  .route("/")
+  .get(asyncHandler(async (req, res) => {
+      const { offset = 0, limit = 10, order = "newest", search = "" } = req.query;
+      let orderBy;
+      switch (order) {
+        case "oldest":
+          orderBy = { createdAt: "asc" };
+          break;
+        case "newest":
+        default:
+          orderBy = { createdAt: "desc" };
+      }
+      const articles = await prisma.article.findMany({
+        where: {
+          OR: [
+            { title: { contains: search, mode: "insensitive" } },
+            { content: { contains: search, mode: "insensitive" } },
+          ],
+        },
+        select: {
+          id: true,
+          title: true,
+          content: true,
+          createdAt: true,
+        },
+        orderBy,
+        skip: parseInt(offset),
+        take: parseInt(limit),
+      });
+      res.status(200).send(articles);
+    }))
+  .post(asyncHandler(async (req, res) => {
+      assert(req.body,CreateArticle);
+      const articles = await prisma.article.create({
+        data: req.body,
+      });
+      res.status(201).send(articles);
+    }))
+
+articleRoute
+  .route("/:id")
+  .get(asyncHandler(async (req, res) => {
+      const { id } = req.params;
+      const articles = await prisma.article.findUnique({
+        where: { id },
+      });
+      res.status(201).send(articles);
```

Comment:
get으로 데이터를 조회 할 때 상태코드는 200을 사용해 주세요.
```suggestion
      res.status(200).send(articles);
```